ozaria/site/components/avatar-selector/PageAvatarSelector/index.vue

```diff
@@ -0,0 +1,216 @@
+
+<script>
+import LayoutCenterContent from '../../common/LayoutCenterContent.vue'
+import LayoutAspectRatioContainer from '../../common/LayoutAspectRatioContainer.vue'
+import LayoutChrome from '../../common/LayoutChrome.vue'
+import Surface from '../../char-customization/common/Surface.vue';
+import BaseButton from '../../common/BaseButton.vue';
+
+// TODO migrate api calls to the Vuex store.
+import { getThangTypeOriginal } from '../../../../../app/core/api/thang-types'
```

Comment:
I've been wondering something lately, as more and more has been moved to the `/orzaria` folder from the `/app` folder. When we use relative paths, especially with many levels like here, we can't actually visually confirm that it is going to the right location at a glance. 

I began replacing these with absolute paths, and found that it was much easier to reason about as I was reading the code. Even for relative paths that are close by, like at the top of this file, it was helpful to visually see the `ozaria/something/something` path. 

What are your thoughts on this? Definitely an eng sync topic, but this was a good use case for a visually hard-to-identify location because of the multiple `../..` relative path steps.

Reply:
This is an excellent eng sync topic. I like relative paths because VScode works very well with relative paths. I.e. If I move a file, it updates all the relative paths. I would still double check absolute paths.

The IDE also seems to preserve type hinting when it is a relative path, but there must be a way to set it up for absolute paths.